### ISSUE TITLE
🐞 BUG: added API_KEY isn't working

### ISSUE THREAD

**Issue by di-sukharev** [insulineru/ai-commit#4]
neither `--apiKey <key>` nor `.env OPENAI_API_KEY=<key>` work.

when using `--apiKey` I get this error:
```
ChatGPTError: ChatGPT error 401: {
    "error": {
        "message": "Incorrect API key provided: true. You can find your API key at https://platform.openai.com/account/api-keys.",
        "type": "invalid_request_error",
        "param": null,
        "code": "invalid_api_key"
    }
}
```

when using `.env OPENAI_API_KEY=<key>` I get this error:
```
Please set the OPENAI_API_KEY environment variable.
```

I didn't see you using `dotenv` lib to read `process.env.OPENAI_API_KEY`, if it's not intended — use `npm i dotenv`

**Comment by di-sukharev** [comment]
I can do the PR — if you want some pair of hands

**Comment by insulineru** [comment]
@di-sukharev variable problem fixed in the 2.0.4 release. Thanks for the report :)
